lib/frameworks/debugprint.js

```diff
@@ -10,10 +9,10 @@ var logger = new Logger('debugger');
  *
  * @param {Runner} runner The current Protractor Runner.
  * @param {Array} specs Array of Directory Path Strings.
- * @return {q.Promise} Promise resolved with the test results
+ * @return {Promise} Promise resolved with the test results
```

Comment:
Since we're getting rid of jshint, shouldn't these files also be moved to TypeScript?

Reply:
Hmmm.... that makes sense. I'll create an issue for myself.

Reply:
https://github.com/angular/protractor/issues/5061